zerver/lib/user_groups.py

```diff
@@ -561,15 +561,15 @@ def get_setting_value_for_user_group_object(
 
 def get_group_setting_value_for_register_api(
```

Comment:
Maybe we can rename this function since it is used at other places as well for stream settings.

Reply:
Yeah, but I'm not sure what to call it. Maybe a good #backend conversation.